packages/transpiler/src/structures/class_implementation.ts

```diff
@@ -50,16 +50,17 @@ export class ClassImplementationTranspiler implements IStructureTranspiler {
         break;
       }
 
-      const intf = scope.findInterfaceDefinition(i.name);
+      const intf = traversal.findInterfaceDefinition(i.name, scope);
```

Comment:
After this change we have problems with escaping the last constant used in our class. 
![obraz](https://user-images.githubusercontent.com/9119187/225577949-efcf9a63-a7f9-488d-abe5-e9b948f3d689.png)